pygloo/main.cc

```diff
@@ -127,6 +127,33 @@ PYBIND11_MODULE(pygloo, m) {
       .def("setTimeout", &gloo::Context::setTimeout)
       .def("getTimeout", &gloo::Context::getTimeout);
 
+
+class FixedBuffer {
```

Comment:
We better move this to a single class file.

Reply:
It seems all classes are edited in this file currently. Maybe we should refine them in another PR which does the cleaning stuffs.